src/terms.js

```diff
@@ -132,11 +132,13 @@ export default [
     'flexibility',
     'foot view',
     'four pillars',
+    'frameworks', 
```

Comment:
Not sure we should include this one because it can be used in a lot of non-bullshit contexts.